pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/PulsarAdminTool.java

```diff
@@ -367,4 +331,49 @@ static void resetLastExitCode() {
         lastExitCode = Integer.MIN_VALUE;
     }
 
+    protected void initJCommander() {
```

Comment:
not sure. It's a init method, not a reset one because the final state (after the method execution) is ready and not empty (after a reset method I expect a non initialized state)